src/Window/Context.cs

```diff
@@ -10,7 +9,7 @@ namespace SFML.Window
     /// This class defines a .NET interface to an SFML OpenGL Context
     /// </summary>
     //////////////////////////////////////////////////////////////////
-    public class Context : CriticalFinalizerObject
```

Comment:
When looking for a replacement for this in .Net Core, this was the best next thing available, because you still need to release the resources